tools/override/main.go

```diff
@@ -139,6 +140,28 @@ func getValue(value string, kf *keyfile.KeyFile, section0 string) string {
 	return value
 }
 
+func mapToSortedList(basket map[string]string) []struct {
+	key string
```

Comment:
Should we define this {key,val} struct some where so that we can void keep define it inline?

Reply:
ok, changed